### PR TITLE
fix: fix feedback 2-jun

### PR DESCRIPTION
**Pull request by asaadam**
Fix all feedbacks from this sheet : 

https://docs.google.com/spreadsheets/d/1hJY24OMi5Zt-z-mhOoZJItP6LTu5IhoBmTzQ8BjBoXE/edit#gid=1331233554

### CODE REVIEW

File: src/utils/config.ts [hunk 1]
```diff
@@ -56,6 +56,21 @@ export const config = {
       ADDRESS: "0x8bdd3d5b889f3d0d735eb4db5d87782df2b4647d",
       ABI: cellarV0816,
     },
+    UNI: {
```
Comment (codingki) [inline src/utils/config.ts:59]: move this upper join it with other tokens

File: src/components/_cards/ApyPerfomanceCard.tsx
Comment (joshuanatanielnm) [inline src/components/_cards/ApyPerfomanceCard.tsx:1]: I think this Typo

File: src/data/actions/common/getUserDataAllStrategies.ts
Comment (joshuanatanielnm) [inline src/data/actions/common/getUserDataAllStrategies.ts:1]: remove console.log, and use console.error for error msg

**Review by joshuanatanielnm — CHANGES_REQUESTED** [review]
Please remove console.log, and update typo text

**Comment by sandbochs** [comment]
Just finished 24hrs of traveling. Will review this in about 8 hrs.

**Pull request merged**